Scala/Scala.sublime-syntax

```diff
@@ -182,21 +182,31 @@ contexts:
           set: annotation-parameters
     - match: '(@)({{alphaplainid}})'
       captures:
-        1: meta.annotation.scala punctuation.definition.annotation.scala
-        2: meta.annotation.identifier.scala variable.annotation.scala
+       1: meta.annotation.scala punctuation.definition.annotation.scala
+       2: meta.annotation.identifier.scala variable.annotation.scala
       push: annotation-parameters
 
   annotation-parameters:
     - meta_content_scope: meta.annotation.parameters.scala
+    - match: (?=\(\))
+      push:
+      - match: \(
+        scope: punctuation.section.arguments.annotation.begin.scala
+      - match: (?=\))
+        scope: punctuation.section.arguments.annotation.end.scala
```

Comment:
Applying a scope to a lookahead doesn't do anything. Or at least, it *shouldn't*. There might be bugs here.

Reply:
It does one thing - escapes from case where is there is no annotation's params list.

Reply:
Well, I mean the `scope` directive doesn't do anything, because the `match` doesn't consume anything.